receiver/rabbitmqreceiver/testdata/integration/scripts/setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
+USER='otel'
+ROOT_PASS='otel'
```

Comment:
nit: Can we set these to different values, to ensure we are not crossing them internally?
```suggestion
USER='otelu'
ROOT_PASS='otelp'
```